src/effgen.ml

```diff
@@ -168,6 +168,8 @@ module StaticGenerators = struct
             [ (* Generate no alphas *)
               (4, Gen.oneofl base_types);
               (1, Gen.map (fun t -> List t) (recgen (n / 2)));
+              (1, Gen.map (fun t -> Option t) (recgen (sqrt n)));
+              (1, Gen.map (fun t -> Ref t) (recgen (sqrt n)));
```

Comment:
It's not necessarily wrong, but I'm not sure why the size for Option/Ref decreases with `sqrt n` while the size for `List` decreases with `n / 2`. Do you know what led to this choice?